src/components/ButtonOption/styled/text.js

```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components';
+import { FONT_SIZE_HEADING_SMALL } from '../../../styles/fontSizes';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import { COLOR_WHITE, COLOR_GRAY_3, COLOR_DARK_1 } from '../../../styles/colors';
+import { lighten } from '../../../styles/helpers/color';
+
+const StyledText = attachThemeAttrs(styled.span).attrs(props => {
+    if (props.palette.isDark) {
+        return {
+            inverse: {
+                text: COLOR_DARK_1,
+                active: lighten(COLOR_DARK_1, 0.6),
+                border: COLOR_DARK_1,
+                disabled: lighten(COLOR_DARK_1, 0.6),
+            },
+        };
+    }
+    return {
+        inverse: {
+            text: COLOR_WHITE,
+            active: COLOR_GRAY_3,
+            border: COLOR_WHITE,
+            disabled: COLOR_GRAY_3,
+        },
+    };
+})`
```

Comment:
I think we should not do this here, it was needed in another component for some transparency issues but here I think we should pass fixed colors from theme, talk to @TahimiLeonBravo about this if you have doubt